time stamp#[s]
float64 velocity_left#[m/s]
float64 velocity_right#[m/s]
float64 rotation_speed_left#[rpm]
float64 rotation_speed_right#[rpm]
float64 velocity#[m/s]
float64 velocity_ratio#[left/right]
float64 angular_velocity#[rad/s]
float64 turning_radius#[m]
float64 curvature#[1/m]
